PSKoans/Koans/dbatools/AboutBackupDatabases.ps1

```diff
@@ -0,0 +1,167 @@
+#Requires -Modules dbatools
```

Comment:
Maybe change the filename to `AboutDatabaseBackups`?

Also, these should all be `.Koans.ps1` as well for consistency.

Reply:
Changed!